enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -9,15 +9,38 @@ import { saveDomImageStyles } from "metabase/visualizations/lib/save-chart-image
  * even when rendered under a React portal.
  */
 export const PublicComponentStylesWrapper = styled.div`
+  // Try to reset as much as possible to avoid css leaking from host app to our components
+  all: initial;
+  text-decoration: none;
+
+  // # Basic css reset
```

Comment:
I was very sad when I realized that we can't "simply" copy paste the css reset and have it scoped to our components 🙃

Reply:
I see, is that because some selectors will leak outside of the PublicComponentStylesWrapper component?